supports/integrate-test/test_mover.py

```diff
@@ -180,6 +180,48 @@ def test_allssd_archive_64MB(self):
         self.assertTrue(cmd_move_1['state'] == "DONE")
         self.assertTrue(cmd_move_2['state'] == "DONE")
 
+    def test_random_list_mover_10MB(self):
+        file_path = "/test/test_file_10MB"
+        cid = create_file(file_path,10*1024)
+        check_storage(file_path)
+
+        # use a list to save the result
+        cid_list = []
+        av_index = -1
+
+        # get the random test list
+        task_list = get_random_task_list(file_path,1000)
+
+        while av_index < 1000:
+            av_index = av_index + 1
+            cid_list[av_index] = task_list[av_index]
+
+        # check the result
+        for i in range(1000):
```

Comment:
Note that Python can return multiple values. So, package all duplicate code into util.py.